episodes/contact-matrices.Rmd

```diff
@@ -0,0 +1,228 @@
+---
+title: 'Contact matrices'
+teaching: 10
+exercises: 2
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What is a contact matrix?
+- How are contact matrices estimated?
+- How are contract matrices used?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Use the R package `socialmixr` to estimate a contact matrix
+- Understand the different types of analysis contact matrices can be used for 
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+<!-- ::::::::::::::::::::::::::::::::::::: prereq -->
+
+<!-- ## Prerequisites -->
+
+
+
+<!-- ::::::::::::::::::::::::::::::::: -->
+
+
+## Introduction
+
+Heterogeneity of contact patterns between different groups affects disease transmission. The rate of contact within and between groups can be presented in a contact matrix. In this tutorial we are going to learn how contact matrices can be used in different analyses and how the `{socialmixr}` package can be used to estimate contact matrices. 
+
+
+```{r,message=FALSE,warning=FALSE}
+library(socialmixr)
+```
+
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+
+
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+## The contact matrix
+
+A contact matrix represents the amount of contact or mixing within and between subgroups. The subgroups are often age categories but can also be geographic areas or high/low risk groups. For example, a contact matrix representing the number of contacts per day in children and adults could be:
+
+$$
+\begin{bmatrix}
+8 & 2\\
+2 & 7 
+\end{bmatrix}
+$$
+Children contact 8 other children each day, adults contact 7 other adults and children contact 2 adults (and vice versa). We can use this kind of information to account for the role heterogeneity in contact plays in infectious disease transmission. 
```

Comment:
This might seem an overly complex point at this stage but I think it would be helpful to clarify here if these are averages or total numbers (could also use this to subtly introduce asymmetry). E.g. we could say

```suggestion
A contact matrix represents the amount of contact or mixing within and between subgroups. The subgroups are often age categories but can also be geographic areas or high/low risk groups. For example, a contact matrix representing the average number of contacts per day between children and adults could be:

$$
\begin{bmatrix}
2 & 2\\
1 & 3 
\end{bmatrix}
$$
In this example, we would use this to represent that children meet, on average, 2 other children and 2 adult per day (first row), and adults meet, on average, 1 child and 3 other adults per day (second row). We can use this kind of information to account for the role heterogeneity in contact plays in infectious disease transmission.